tools/gmhazard_scripts/db_creation/empirical_db/new_calc_emp_ds.py

```diff
@@ -55,6 +54,8 @@ def create_rupture_context_df(
     # if there is only a single point then the dtop, dbot and hypo_depth
     # are at the same point
     rupture_df[["hypo_depth", "ztor"]] = rupture_df[["dbot", "dtop"]]
```

Comment:
Hmmm, this assumption doesn't seem like it's being applied correctly.

Reply:
Yeah I'm slightly confused by this line?

Reply:
@jasonmotha Do we treat all DS sources as point sources?

Reply:
Yep.